website/docs/d/elasticache_replication_group.html.markdown

```diff
@@ -32,6 +32,8 @@ In addition to all arguments above, the following attributes are exported:
 * `arn` - ARN of the created ElastiCache Replication Group.
 * `auth_token_enabled` - Whether an AuthToken (password) is enabled.
 * `automatic_failover_enabled` - A flag whether a read-only replica will be automatically promoted to read/write primary if the existing primary fails.
+* `ip_discovery` - The IP version advertised in the discovery protocol.
```

Comment:
Can you remove the changes you made in the data source docs, you haven't actually updated the data sources. Lets do that through a different PR.

Reply:
Completed